### PR TITLE
Setting up Sphinx and preparing readthedocs.io

### LINKED ISSUE #820: Set up Sphinx
I started setting up Sphinx some time back but haven't had the time to get back to it.
Anybody who want to work on this should have a look at https://github.com/mozilla/OpenWPM/commit/2e17baa15e59e5d49549f1d59f3c1ad868995bd9

### PR DESCRIPTION
**Pull request by jhabarsingh**
Resolved Issue 820
Fixes #820

### CODE REVIEW

**Comment by vringar** [comment]
@jhabarsingh Did you do both steps described [here](https://github.com/mozilla/OpenWPM/blob/master/CONTRIBUTING.md#setting-up-a-dev-enviroment)
You PR is currently failing because the code changes are not what our tooling allows.
Installing all dependencies and running `pre-commit run --all` should fix this.

**Comment by vringar** [comment]
Also please run `repin.sh` in the `scripts` folder as described [here](https://github.com/mozilla/OpenWPM/blob/master/CONTRIBUTING.md#managing-requirements)

**Comment by jhabarsingh** [comment]
I did run `resin.sh` before sending you the PR.

**Comment by jhabarsingh** [comment]
commit [`131a002`](https://github.com/mozilla/OpenWPM/pull/834/commits/131a00279633a0c06cf50f846fec2cd2afda8dbd).
When I installed `nbsphinx` it changed `environment.yaml` file so therefore I changed it to the default one. 

That's what this commit is all about.

**Comment by vringar** [comment]
Please add the changed `enviroment.yaml`
What we mean by not changing `enviroment.yaml` directly.

Our process for adding new dependencies is:
1. Add it to `scripts/enviroment-unpinned.yaml`
2. Run `scripts/repin.sh`
3. Commit the changed `scripts/enviroment-unpinned.yaml` and `environment.yaml`

**Comment by jhabarsingh** [comment]
I've changed the `environment.yaml` file.

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+version: 2
+
```
Comment (vringar) [inline .readthedocs.yml:2]: I think as per [this paragraph](https://docs.readthedocs.io/en/stable/config-file/v2.html#sphinx)  we also need to specify where our `conf.py` file is located.

**Review by vringar — CHANGES_REQUESTED** [review]

File: .readthedocs.yml (hunk 1 above)
Comment (jhabarsingh) [reply .readthedocs.yml:3]: Sir I have specified the location of conf.py.

**Pull request closed without merging**